src/__tests__/set-a11y-status.js

```diff
@@ -5,38 +5,45 @@ beforeEach(() => {
 })
 
 test('sets the status', () => {
-  const setA11yStatus = setup()
+  const {default: setA11yStatus} = setup()
   setA11yStatus('hello')
   expect(document.body.firstChild).toMatchSnapshot()
 })
 
+test('removes the status div', () => {
+  const {default: setA11yStatus, removeStatusDiv} = setup()
+  setA11yStatus('hello')
+  removeStatusDiv()
+  expect(document.body.firstChild).toBe(null)
+})
+
 test('replaces the status with a different one', () => {
-  const setA11yStatus = setup()
+  const {default: setA11yStatus} = setup()
```

Comment:
this is a bit ugly, but I don't know how to import the default and the named export with require.